### PR TITLE
[FLINK-34677][cdc] Synchronize Chinese documentations for Flink CDC

### PR DESCRIPTION
**Pull request by PatrickRen**
This pull request synchronized Chinese documentations for Flink CDC, to fully match English docs. Translations will be made by followup PRs.

### CODE REVIEW

File: docs/content.zh/docs/get-started/quickstart/mysql-to-doris.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ under the License.
 
 # Streaming ELT from MySQL to Doris
 
-This tutorial is to show how to quickly build a Streaming ELT job from MySQL to Doris using Flink CDC, including the
+This tutorial is to show how to quickly build a Streaming ELT job from MySQL to StarRocks using Flink CDC, including the
```
Comment (gtk96) [inline docs/content.zh/docs/get-started/quickstart/mysql-to-doris.md:29]: For what reason have modifications been made here?

Comment (PatrickRen) [reply docs/content.zh/docs/get-started/quickstart/mysql-to-doris.md:29]: Good catch! I just copy all documentations from `docs/content/docs` to `docs/content.zh/docs`. We need to trace back to the original author

Comment (leonardBang) [reply docs/content.zh/docs/get-started/quickstart/mysql-to-doris.md:29]: +1

> For what reason have modifications been made here?

File: docs/content.zh/docs/developer-guide/contribute-to-flink-cdc.md [hunk 2]
```diff
@@ -24,56 +24,61 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-# 社区贡献
+# Contributing
```
Comment (leonardBang) [inline docs/content.zh/docs/developer-guide/contribute-to-flink-cdc.md:27]: Could we keep this Chinese doc instead fo override it?

File: docs/content.zh/docs/developer-guide/licenses.md [hunk 3]
```diff
@@ -24,15 +24,15 @@ specific language governing permissions and limitations
 under the License.
 -->
```
Comment (leonardBang) [inline docs/content.zh/docs/developer-guide/licenses.md:25]: Could we keep this Chinese doc instead fo override it?

**Review by leonardBang — COMMENTED** [review]
Thanks @PatrickRen for the contribution, I left two minor comments

File: docs/content.zh/docs/get-started/quickstart/mysql-to-doris.md (hunk 1 above)
Comment (leonardBang) [reply docs/content.zh/docs/get-started/quickstart/mysql-to-doris.md:29]: > Good catch! I just copy all documentations from `docs/content/docs` to `docs/content.zh/docs`. We need to trace back to the original author

Looks like I made the typo, o(╯□╰)o

Comment (leonardBang) [reply docs/content.zh/docs/get-started/quickstart/mysql-to-doris.md:29]: Please help fix it，sir

Comment (PatrickRen) [reply docs/content.zh/docs/get-started/quickstart/mysql-to-doris.md]: Fixed in the followup commit.

**Comment by gtk96** [comment]
@PatrickRen Outstanding contribution, LGTM

**Comment by gtk96** [comment]
@leonardBang https://github.com/apache/flink-cdc/blob/master/docs/content/docs/get-started/quickstart/mysql-to-doris.md  We also need a small patch here

**Comment by PatrickRen** [comment]
@gtk96 It has been fixed in my followup commit [2646d2f](https://github.com/apache/flink-cdc/pull/3169/commits/2646d2faebe5e17beb193a2e047480e490883155)

**Review by gtk96 — APPROVED** [review]

**Pull request merged**